src/interface/viewport.c

```diff
@@ -1331,7 +1331,7 @@ bool sub_68818E(uint32 image_id, uint16 x, uint16 y, paint_struct ** paint)
 		return false;
 	}
 
-	RCT2_GLOBAL(0x00EE7888, uint32) += 0x12;
+	unk_EE7888 = (paint_struct *)((uintptr_t)unk_EE7888 + 0x12);
```

Comment:
EE7888 is a pointer to the next available paint struct. There are 3 different types of paint struct with 3 different sizes. If you have a look at https://github.com/OpenRCT2/OpenRCT2/pull/3438 we are making this all more obvious. I'm not sure what you want to do for the x64 port stuff.